README.md

```diff
@@ -1,22 +1,22 @@
-# Upload Feature for AdminBro
+# Upload Feature for AdminJS
```

Comment:
Can you update to `Password Feature`? The old readme was incorrect

Reply:
Done